test/browser/hooks/useEffect.test.js

```diff
@@ -32,13 +32,12 @@ describe('useEffect', () => {
 			return null;
 		}
 
-		render(<Comp />, scratch);
-		render(<Comp />, scratch);
-
-		expect(cleanupFunction).to.be.not.called;
-		expect(callback).to.be.calledOnce;
+		act(() => {
+			render(<Comp />, scratch);
+			render(<Comp />, scratch);
+		});
 
-		render(<Comp />, scratch);
```

Comment:
I'm honestly scared because act is now a hard requirement for useEffect which it wasn't before... This could break some tests :(

Reply:
Yea, agreed